.changeset/rare-worms-vanish.md

```diff
@@ -0,0 +1,9 @@
+---
+"@lynx-js/react-rsbuild-plugin": minor
+---
+
+**Breaking Changes**:
```

Comment:
Does this need to be a **BREAKING CHANGE**?

We didn't make it a **BREAKING CHANGE** before when changing to `'jsReady'` in https://github.com/lynx-family/lynx-stack/pull/883.

https://github.com/lynx-family/lynx-stack/pull/883/files#diff-e6cc62bac385f8ba6331777e7201c202a86d38d51b26934be11a3aed70907abdR212

And it turns out that it works fine until now. So I don't think it's necessary to be a breaking change when revert back.

Reply:
I was thinking that people using old version of `@lynx-js/template-webpack-plugin` and `@lynx-js/webpack-runtime-globals`.

If they update only `@lynx-js/react-rsbuild-plugin` they may break because they rely on 'jsReady' to work properly.

What do you think?

Reply:
Let me change to patch first, I think user can still config 'jsReady' manually if the above case happens.